package.json

```diff
@@ -4,8 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec ts-node src/index.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon --watch './**/*.ts' --exec ts-node src/index.ts",
+    "seed": "npx ts-node prisma/seed.ts",
+    "lint": "tsc --noEmit && eslint src --ext \".ts\"",
```

Comment:
Mmm, I would remove this line and kept only the one below with the name 'lint' because it's the standard configuration that we use in the templates 😂

Reply:
Fixed